deploy/examples/common.yaml

```diff
@@ -548,6 +548,18 @@ rules:
 ---
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-rgw
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list", "create", "patch", "update", "delete"]
```

Comment:
RGW is only the ceph daemon with no rook operator in the image, right? Is rook being added to the rgw image in the future or why are these privs needed for configmaps and secrets? The rgw daemon won't be accessing them directly, right?

Reply:
@travisn: currently Rook operator `PodSpec` for RGW which have configmap/secrets to need attach or create hence I have created this Rules. If it is not required then I will remove it. I was not 100% sure this role is needed. So most likely I need to create SA without any roles/rolebinding. Am I right??

Reply:
@thotz The pod actually doesn't need RBAC for the confimaps/secrets that are being mounted to the pod, it only needs the rbac if it is going to access the K8s API from a running container. Just creating the service with no roles/bindings sounds good.